tools/android_support/files.json

```diff
@@ -2,41 +2,64 @@
   {
     "url": "https://dl.google.com/dl/android/maven2/android/arch/lifecycle/common/1.1.1/common-1.1.1.jar",
     "out_file_name": "android_arch_lifecycle_common.jar",
-    "maven_dependency": "android.arch.lifecycle:common:1.1.1"
+    "maven_dependency": "android.arch.lifecycle:common:1.1.1",
+    "provides": [
+      "android.arch.lifecycle.Lifecycle",
+      "android.arch.lifecycle.LifecycleObserver",
+      "android.arch.lifecycle.LifecycleOwner"
+    ]
   },
   {
     "url": "https://dl.google.com/dl/android/maven2/android/arch/lifecycle/common-java8/1.1.1/common-java8-1.1.1.jar",
     "out_file_name": "android_arch_lifecycle_common_java8.jar",
-    "maven_dependency": "android.arch.lifecycle:common-java8:1.1.1"
+    "maven_dependency": "android.arch.lifecycle:common-java8:1.1.1",
+    "provides": [
+      "android.arch.lifecycle.DefaultLifecycleObserver"
+    ]
   },
   {
     "url": "https://dl.google.com/dl/android/maven2/android/arch/lifecycle/runtime/1.1.1/runtime-1.1.1.aar",
     "out_file_name": "android_arch_lifecycle_runtime.jar",
-    "maven_dependency": "android.arch.lifecycle:runtime:1.1.1"
+    "maven_dependency": "android.arch.lifecycle:runtime:1.1.1",
+    "provides": [
+      "android.arch.lifecycle.LifecycleRegistry"
+    ]
   },
   {
     "url": "https://dl.google.com/dl/android/maven2/android/arch/lifecycle/viewmodel/1.1.1/viewmodel-1.1.1.aar",
     "out_file_name": "android_arch_lifecycle_viewmodel.jar",
-    "maven_dependency": "android.arch.lifecycle:viewmodel:1.1.1"
+    "maven_dependency": "android.arch.lifecycle:viewmodel:1.1.1",
+    "provides": []
```

Comment:
does this mean this can be deleted too?

Reply:
these are still needed to compile the embedding. This is a transitive dependency of support-fragment.

Reply:
I see, you're saying this ends up in the user's APK either way ultimately?

Reply:
correct. Gradle will bring this dependency anyways.

Reply:
understood, thanks